Packs/iDefense/Integrations/iDefense_v2/iDefense_v2.py

```diff
@@ -0,0 +1,274 @@
+from typing import Union
+
+from CommonServerPython import *
+
+'''IMPORTS'''
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+'''CONSTANTS'''
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+class Client(BaseClient):
+
+    def __init__(self, input_url: str, api_key: str, verify_certificate: bool, proxy: bool):
+        base_url = urljoin(input_url, '/rest/threatindicator/v0')
+        headers = {
+            "Content-Type": "application/json",
+            'auth-token': api_key
+        }
+        super(Client, self).__init__(base_url=base_url,
+                                     verify=verify_certificate,
+                                     headers=headers,
+                                     proxy=proxy)
+
+    def perform_request(self, name: str, value: str):
+        data = {}
+        if value:
+            data = {'key.values': value}
```

Comment:
you can pass data as an arg with a default value, and in case of need feel it, i'll comment in a func for example

Reply:
Ok, I change it and then I saw that with this situation this function maybe not necessary? It looks like this:
    def threat_indicator_search(self, url_suffix: str, data: dict = {}) -> dict:
        return self._http_request(method='GET', url_suffix=url_suffix, params=data)

Reply:
please still keep it, it is a better practice to perform the `_http_request` from within a client function.
https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles